### ISSUE TITLE
Does dag-factory support PythonVirtulenvOperator?

### ISSUE THREAD

**Issue by gdpark14** [ajbosco/dag-factory#133]
I wonder this library supports PythonVirtualenvOperator... I guess it doesn't.

**Comment by pankajastro** [comment]
Yes, it is supported and expected to work as long as Python class for particular operator exist in your env. closing it please reopen with logs if you hit any issue.
